### PR TITLE
Replace is_it_working with okcomputer

### LINKED ISSUE #45: thumbnail service: deploy first pass at okcomputer for monitoring
Update:  there *is* a nagios check on is_it_working in production!   But no fear, I have restored is_it_working for interim deploys as we work on better upgraded monitoring.

because there is currently no nagios check, it is okay to get rid of is_it_working in the same PR

blockedby #48?

### PR DESCRIPTION
**Pull request by lmcglohon**
@drh-stanford @ndushay Fixes #45

### CODE REVIEW

File: config/initializers/okcomputer.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# Simple echo of the VERSION file
+class VersionCheck < OkComputer::AppVersionCheck
+  def version
+    File.read(Rails.root.join('VERSION')).chomp
+  rescue Errno::ENOENT
+    raise UnknownRevision
+  end
+end
+OkComputer::Registry.register 'version', VersionCheck.new
```
Comment (ndushay) [inline config/initializers/okcomputer.rb]: Ditto question in sul-dlss/was-registrar/pull/39  about how we want to do a version check in okcomputer.

File: config/initializers/okcomputer.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require 'okcomputer'
+
```
Comment (ndushay) [inline config/initializers/okcomputer.rb]: I've been mounting okcomputer at "status" but obviously it's a personal preference thing:

```
# /status for 'upness', e.g. for load balancer
# /status/all to show all dependencies
# /status/<name-of-check> for a specific check (e.g. for nagios warning)
OkComputer.mount_at = 'status'
OkComputer.check_in_parallel = true
```

File: Gemfile [hunk 3]
```diff
@@ -23,7 +23,7 @@ gem 'daemons' # ruby code can be run as daemon with simple start/stop/restart co
 gem 'whenever', :require => false
 
 gem 'druid-tools'
-gem 'is_it_working-cbeer'
+gem 'okcomputer'
```
Comment (ndushay) [inline Gemfile]: we can only remove is_it_working in the same PR if  nagios isn't expecting anything at `is_it_working` path.

**Comment by ndushay** [comment]
any additional dependencies on other stuff (urls, directories, whatever) that should be in status/all page?

**Comment by lmcglohon** [comment]
We are going to remove the version check and add the external dependency checks.

**Comment by lmcglohon** [comment]
@ndushay Just need to determine external dependencies for alerting which is covered by related issue #53.  I believe this is OK to merge.

**Comment by ndushay** [comment]
ah - so adding the external dependencies with be a separate PR?

**Pull request merged**